store/postgres/src/chain_store.rs

```diff
@@ -607,7 +610,7 @@ mod data {
             first_block: i64,
             hash: H256,
             genesis: H256,
-        ) -> Result<Vec<H256>, Error> {
+        ) -> Result<Option<H256>, Error> {
```

Comment:
Should `missing_parents` be renamed to `missing_parent`?

Reply:
Totally. Fixed